docs/pages/enterprise/sso/oidc.mdx

```diff
@@ -23,14 +23,37 @@ administrators to define policies like:
 ## Enable OIDC Authentication
 
 First, configure Teleport auth server to use OIDC authentication instead of the local
-user database. Update `/etc/teleport.yaml` as show below and restart the
-teleport daemon.
+user database. For self-hosted update `/etc/teleport.yaml` and restart the teleport daemon.  
```

Comment:
Maybe move the "For self-hosted update" instruction into the relevant `TabItem`, since it's part of the workflow specific to that deployment type.